status/status_test.go

```diff
@@ -144,6 +145,37 @@ func (s) TestFromErrorOK(t *testing.T) {
 	}
 }
 
+func (s) TestFromErrorWrapped(t *testing.T) {
+	code, message := codes.Internal, "test description"
```

Comment:
Nit:
```
const code, message = codes.Internal, "test description"
```

(to avoid these being mutable variables when they should be invariants.)

And the other test cases, below.